.github/workflows/deploy_to_firebase_app_distribution.yml

```diff
@@ -0,0 +1,69 @@
+name: Deploy staging to Firebase App Distribution
+
+on:
+  # Trigger the workflow on push action in develop branch.
+  # So it will trigger when the PR of the feature/chore/bugfix branch was merged.
+  push:
+    branches:
+      - develop
```

Comment:
@hoangnguyen92dn my suggestion to test this workflow is adding `chore/2-setup-cicd` into this filter and removing it after done 👌

Reply:
@luongvo I checkout the test branch from this chore branch and update the pattern to `'test/**'`. You can check the workflow executed [here](https://github.com/nimblehq/jetpack-compose-crypto/actions/runs/2737014846/workflow) 😄